profile/README.md

```diff
@@ -1,36 +1,60 @@
 
-### Key Features:
+---
+
+## 🔎 What's in the FTW Ecosystem?
+
+### 🗂️ FTW Benchmark Dataset  
+Originally the core of the initiative, the **FTW Benchmark Dataset** is a global collection of labeled agricultural field boundaries, designed to benchmark model performance and support open research. It aggregates and harmonizes a number of open datasets into 1.6 million parcel boundaries and over 70,000 samples covering diverse agricultural landscapes across 4 continents and 24 countries.
+
+**What’s included in the Benchmark Dataset?**  
+- Publicly available, labeled field boundaries from around the world  
+- Uniform data formatting and metadata schema  
+- A maintained subset of training/validation/test data  
+- Tools to evaluate model performance on shared benchmarks  
+- Contribution guidelines for submitting new datasets
+
+
+### 🧠 FTW Baseline Models  
+Reference implementations of machine learning models trained on the FTW Benchmark Dataset. These models offer starting points and performance baselines for researchers and practitioners.
 
-1. **Near-Global Coverage**: FTW spans four continents—Europe, Africa, Asia, and South America—covering diverse agricultural landscapes across 24 countries. This extensive geographic coverage allows for the development of models that can generalize well to different agricultural practices and field types.
+### 🛠️ FTW Tools  
+A growing suite of tools to support end-to-end workflows:
+- **Data access tools**: Python libraries and CLI tools to download and work with the benchmark dataset  
+- **Inference tools**: Scripts and utilities to run field boundary detection on new imagery  
+- **Formatting tools**: Resources to convert raw data into FTW-compliant format (e.g., fiboa to FTW train/test split)  
+- **QGIS Plugin**: A plugin to browse and interact with field data within QGIS
 
-2. **Large-Scale Dataset**: With approximately 1.6 million parcel boundaries and over 70,000 samples, FTW is significantly larger than previously available datasets. Each sample includes instance and semantic segmentation masks paired with multi-date, multi-spectral Sentinel-2 satellite images, enabling detailed temporal and spectral analysis.
+### 🌐 FTW Web App  
+A browser-based interface to explore, select, and analyze scenes using FTW models and data. 
 
-3. **Multi-class Segmentation**: The dataset provides masks for both instance segmentation and semantic segmentation with different classes, including:
-   - **Instance Segmentation Masks**: To identify individual fields.
-   - **Semantic Segmentation Masks**: 
-     - Two-class masks: Background and polygon (field).
-     - Three-class masks: Background, polygon (field), and boundaries.
+### 📡 FTW Source Data  
+Curated public datasets of manually labeled field boundaries—such as those hosted on [fiboa.org](https://fiboa.org)—that may serve as inputs to the benchmark dataset. These are raw sources prior to FTW formatting.
```

Comment:
I don't think we want to bring up that they're hosted on fiboa.org. But we should mention that they are formatted using fiboa. 

```suggestion
Curated public datasets of field boundaries from government sources and hand-labeling campaigns, that may serve as inputs to the benchmark dataset. These are converted to [fiboa](https://fiboa.org) but are prior to FTW subsetting and harmonization.
```